### PR TITLE
#6 Fix the E2E test

### PR DESCRIPTION
**Pull request by lukasz-gryzbon**
- fix the diagrams generated by the e2e test
- use the `lsd-formatting-library`
- add examples to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,40 @@
...
+# Usage
+
+To use the `lsd-distributed-interceptors` library just add it to the production dependencies:
+
+```groovy
+implementation "io.github.lsd-consulting:lsd-distributed-interceptor-library:1.0.0"
+```
+
+and configure through environment properties:
+
```
Comment (nickmcdowall) [inline README.md:33]: Wonder if it is worth saying `app properties`? (environment properties made me think environment variables at first).

**Review by nickmcdowall — APPROVED** [review]
👍

File: README.md (hunk 1 above)
Comment (lukasz-gryzbon) [reply README.md:33]: I agree 👍

**Pull request merged**